core/src/kmx/kmx_processevent.cpp

```diff
@@ -11,7 +11,7 @@ using namespace kmx;
 
 /* Globals */
 
-KMX_BOOL km::core::kmx::g_debug_ToConsole = FALSE;
+KMX_BOOL km::core::kmx::g_debug_ToConsole = TRUE;
```

Comment:
```suggestion
KMX_BOOL km::core::kmx::g_debug_ToConsole = FALSE;
```

We don't want this on in a release build. It's debug only.

Reply:
🤦 this was supposed to be git skipped.